playwright-e2e/dsm/component/tables/onc-history-table.ts

```diff
@@ -0,0 +1,303 @@
+import {expect, Locator, Page} from '@playwright/test';
+import Table from 'dss/component/table';
+import DatePicker from '../date-picker';
+import TextArea from '../../../dss/component/textarea';
+import {
+  InputTypeEnum,
+  OncHistoryInputColumnsEnum,
+  OncHistorySelectRequestEnum
+} from '../tabs/enums/onc-history-input-columns-enum';
+import {OncHistoryInputs} from '../tabs/models/onc-history-inputs';
+import {
+  OncHistoryInputsMapValue,
+  OncHistoryInputsTypes
+} from '../tabs/interfaces/onc-history-inputs-types';
+import {waitForResponse} from '../../../utils/test-utils';
+import Select from '../../../dss/component/select';
+import TissueInformationPage from '../../pages/tissue-information-page/tissue-information-page';
+import Button from '../../../dss/component/button';
+import {FillDate} from '../../pages/tissue-information-page/interfaces/tissue-information-interfaces';
+import Input from '../../../dss/component/input';
+
+export default class OncHistoryTable extends Table {
+  private readonly tissueInformationPage = new TissueInformationPage(this.page);
```

Comment:
Hmm, I saw something else similar like this recently and I had to change because we should instatiate a new object like this outside the constructor.  `tissueInformationPage = new TissueInformationPage(this.page);` should be inside the constructor because `this.page` is created when the constructor is running.

Reply:
I didn't encounter any issues with it, which is why I left it as is. However, I do acknowledge that your suggestion is a better approach, so I'll make the necessary adjustments. Thank you for pointing that out.